src/_playground/documentation/DocsTile/DocsTile.js

```diff
@@ -2,51 +2,62 @@ import { Button } from '../../../';
 import classnames from 'classnames';
 import { googlecode } from 'react-syntax-highlighter/styles/hljs';
 import PropTypes from 'prop-types';
+import React from 'react';
 import SyntaxHighlighter from 'react-syntax-highlighter';
 import { Toggle } from '../../../Toggle/Toggle';
-import React, { Component } from 'react';
 
-export class DocsTile extends Component {
+export class DocsTile extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            backgroundToggle: false
+            backgroundOn: true,
+            rightToLeft: false
```

Comment:
Per comment below, maybe change these state variables to `hideBackground` and `showRTL` so both can be defaulted to `false` (matching the visual "off" state of the toggle).